src/java.desktop/share/classes/javax/swing/text/html/HTMLEditorKit.java

```diff
@@ -1307,6 +1299,27 @@ public void handleEndOfLineString(String eol) {
      *     <td>FrameView
      * </tbody>
      * </table>
+     *
+     * <p>
+     * @implNote
+     * Parsed tags that are unrecognized or are recognized but unsupported are
+     * handled differently by the editor.
+     *
+     * <ul>
+     * <li>When the container using HTML tags is editable:</li>
+     * <ul>
+     *     <li>Tags display the tag text as editable text fields.</li>
+     *     <li>The content within the tags will be handled by the editor as
+     *     regular text.</li>
+     * </ul>
```

Comment:
You should close `</li>` on the first line but after closing `</ul>`: the inner `<ul>` has to be inside a list item.

The list should look like:
```html
<ul>
    <li>Outer 1st
        <ul>
            <li>Inner</li>
        </ul>
   </li>
   <li>Outer 2nd</li>
</ul>
```

Reply:
> You should close `</li>` on the first line but after closing `</ul>`: the inner `<ul>` has to be inside a list item.

`make docs` also reports an error here:

```
...\javax\swing\text\html\HTMLEditorKit.java:1310:
error: tag not allowed here: <ul>
     * <ul>
       ^
```

Reply:
@aivanov-jdk I believe I fixed it. I ran make docs and I got no errors now.